setup.cfg

```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts = --self-contained-html --html=report.html
```

Comment:
There are implementation(s) pending for the report, yeah?
Or should the reports be generating already?

Reply:
The report should be generated when you run `pytest`. You'll find a file called `report.html`

Reply:
Yes, I just checked and this is working.